lib/ui/painting.dart

```diff
@@ -1580,6 +1713,37 @@ enum ColorSpace {
   /// see the extended values an [ImageByteFormat] like
   /// [ImageByteFormat.rawExtendedRgba128] must be used.
   extendedSRGB,
+  /// The Display P3 color space.
+  ///
+  /// This is a wide gamut color space that has broad hardware support,
+  /// especially Apple devices.
```

Comment:
This should document the state of the world on flutter, regardless of hardware support and what happens when it is used on unsupported platforms.

Example: Supported on iOS devices when using the Impeller rendering engine. If used on other platforms, values will be clamped at max sRGB values.

Reply:
Done.  I didn't want to say anything that will go out of date so I mentioned impeller ios as an example and mentioned what happens when it isn't supported.